src/pages/account/signup.tsx

```diff
@@ -157,7 +26,138 @@ type RegistrationFormInput = {
 };
 
 export default function Signup() {
+  const renderForm = (i: number) => {
```

Comment:
I'm not sure about this but is there any reason for putting this function inside of the Signup function. Can you explain a bit?